src/orion/executor/dask_backend.py

```diff
@@ -42,10 +81,31 @@ def wait(self, futures):
         results = self.client.gather(list(futures))
         if self.in_worker:
             rejoin()
+        return [r.get() for r in results]
+
+    def async_get(self, futures, timeout=0.01):
```

Comment:
Side effect should be documented here. `futures` will be modified.

Reply:
It should also be tested in a generic test for async_get.

Reply:
The side effect is documented in the mother. Sphinx should document this method with the mother's docstring in that case

```
"""Retrieve futures that completed, removes them from the list of pending futures
        and return their results

        Parameters
        ----------
        futures: `concurrent.futures.Futures` or equivalent interface
            The objects returned by ``submit()`` of the executor.

        timeout: int
            time to wait before checking the other future

        Returns
        -------
        returns a list fo results

        """
```